buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterConfiguration.groovy

```diff
@@ -131,7 +137,7 @@ class ClusterConfiguration {
      * condition is executed.
      */
     @Input
-    int nodeStartupWaitSeconds = 30
+    int nodeStartupWaitSeconds = 120
```

Comment:
I'd rather not do this - I think there's something wrong with the discovery plugin tests.